src/Microsoft.Diagnostics.Monitoring.RestServer/appsettings.json

```diff
@@ -6,8 +6,11 @@
       "Microsoft.Hosting.Lifetime": "Information"
     },
     "Console": {
-      "IncludeScopes": true,
-      "TimestampFormat": "HH:mm:ss "
+      "FormatterName": "simple",
```

Comment:
Does this format work well with Container Insights?

Reply:
Per PR description, I plan use environment variables in the Docker image to set it to use JSON formatting with additional tweaks and keeping the simple/default format for the tool itself so that when used in non-production environments the logging is readable.

Reply:
The JSON formatting will allow Log Analytics queries to more easily extract logging information (timestamp, level, event ID, category, etc) per logged entry.